nuxeo-audit-query-page-provider.html

```diff
@@ -0,0 +1,284 @@
+<!--
+@license
+(C) Copyright 2015 Nuxeo SA (http://nuxeo.com/) and contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+​
+    http://www.apache.org/licenses/LICENSE-2.0
+​
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="nuxeo-operation.html">
+
+<!--
+`nuxeo-document-history-page-provider` performs a query with DOCUMENT_HISTORY_PROVIDER against audit logs.
+
+    <nuxeo-document-history-page-provider doc-id="..."
+                                          current-page="{{entries}}">
+    </nuxeo-document-history-page-provider>
+
+@group Nuxeo Elements
+@element nuxeo-audit-page-provider
+@homepage http://www.nuxeo.org
+-->
+<dom-module id="nuxeo-audit-query-page-provider">
```

Comment:
Suggestion: use a shorter name for the element like **nuxeo-audit-page-provider** maybe?